tfe/resource_tfe_registry_module_test.go

```diff
@@ -230,6 +230,61 @@ func TestAccTFERegistryModule_publicRegistryModule(t *testing.T) {
 						"tfe_registry_module.foobar", "namespace", expectedRegistryModuleAttributes.Namespace),
 					resource.TestCheckResourceAttr(
 						"tfe_registry_module.foobar", "registry_name", string(expectedRegistryModuleAttributes.RegistryName)),
+					resource.TestCheckResourceAttr(
+						"tfe_registry_module.foobar", "no_code", "false"),
```

Comment:
From what I mentioned above, this never gets set because `no_code` is only ever written to state during `Update()` and `Update()` is only called when the attribute is set as an argument in your configuration.